protocol/README.md

```diff
@@ -25,147 +25,149 @@ Every TBDex message contains the following fields:
 
 The `body` of each message can be any of the following message types
 
-## `RequestForQuote`
-
+## `Offering`
+| field            | data type | required | description                                                                                          |
+| ---------------- | --------- | -------- | ---------------------------------------------------------------------------------------------------- |
+| `id` | string    | Y        | A unique identifier for this offering.|
+| `pair` | string    | Y        | The currency pair being offered, in the format of `basecurrency_countercurrency`.|
```

Comment:
Splitting this into an object, like what @michaelneale, seems like a good approach.

Reply:
so i'm going to actually punt on this and keep it as `pair` for now, with the caveat that this will definitely evolve (just like what we talked about with the `fee` key). i'd like to wrap up the conversation so we can get _something_ written down for initial projects, but we can definitely have this discussion again. also cc'ing @michaelneale since he first suggested this pair be split